tests/console/mysql_odbc.pm

```diff
@@ -0,0 +1,91 @@
+# SUSE's openQA tests
+#
+# Copyright © 2017 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: test that uses isql command line tool to interact with unixODBC
+# plugin
+# This test consists on a MySQL/MariaDB database and a unixODBC plugin. It
+# first inserts an element. Then uses isql to read the elements.
+# If succeed, the test passes, proving that the connection is working.
+#
+# Maintainer: Ednilson Miura <emiura@suse.cz>
+
+use base "consoletest";
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+sub write_odbc {
+    assert_script_run "echo [myodbc_mysql_dsn] > /etc/unixODBC/odbc.ini";
```

Comment:
what about use HEREDOC on this place ?

hint: https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/66a8f9b64d0016e54b6a7f525a3bcf7320048068/tests/kernel/ltp_setup_networking.pm#L31